scss/active.scss

```diff
@@ -0,0 +1,98 @@
+@import "vars";
+.backIcon {
+  width: 24px;
+  height: 18px;
+  position: relative;
+  border: 0;
+  background-color: transparent;
+  cursor: pointer;
+  &::after {
+    content: "";
+    position: absolute;
+    width: 24px;
+    height: 2px;
+    background-color: $colorTextPrimary;
+    left: 0;
+    top: 5px;
+    transform: rotate(45deg);
+  }
+  &::before {
+    content: "";
+    position: absolute;
+    width: 24px;
+    height: 2px;
+    background-color: $colorTextPrimary;
+    left: 0;
+    top: 5px;
+    transform: rotate(135deg);
+  }
+}
+
+.none {
+  display: none;
+}
+
+.rotate {
+  transform: rotate(135deg);
+  transition: transform 0.8s ease-in-out;
+}
+
+.active-link {
+  &::after {
+    content: "";
+    position: absolute;
+    margin-top: 20px;
+    left: 20px;
+    width: 40px;
+    height: 2px;
+    background-color: $colorTextlinks;
+  }
+}
+
+.current-tabs-link {
+  color: $colorTextlinks;
+}
+
+.overflow {
```

Comment:
class name must be informative , reading such class name, it is difficult to understand what for this class was created.
use for ex. `is-hidden` , I have told about class-helpers last meeting

Reply:
done